plugins/TelescopeControl/src/TelescopeClient.cpp

```diff
@@ -125,6 +126,31 @@ TelescopeClient *TelescopeClient::create(const QString &url)
 TelescopeClient::TelescopeClient(const QString &name) : nameI18n(name), name(name)
 {}
 
+
+bool TelescopeClient::GetAltAzFromJ2000Position(const Vec3d &j2000Pos,TelescopeControl::Equinox equinox ,double& alt, double &az) const
+{
+   const StelCore* core = StelApp::getInstance().getCore();
+   const  bool res = core != nullptr;
+   if(res)
+   {
+      const bool equinoxJNow = equinox == TelescopeControl::EquinoxJNow;
+      StelCore::RefractionMode mode = equinoxJNow ? StelCore::RefractionOff : StelCore::RefractionAuto;
```

Comment:
This logic is unclear. What does refraction setting have to do with JDnow? Please add a comment that explains what you are doing here.